tests/bitcoin/BitcoinClient.spec.ts

```diff
@@ -37,26 +41,62 @@ describe('BitcoinClient', async () => {
     });
   }
 
+  describe('generatePrivateKey', () => {
+    it('should construct a PrivateKey and export its WIF', () => {
+      const privateKey = BitcoinClient.generatePrivateKey('mainnet');

```

Comment:
```suggestion
      const privateKey = BitcoinClient.generatePrivateKey('testnet');
```
Not even in tests. Don't try it kids. Satoshi Clients will know.

Reply:
Good catch. Done.